client/src/components/ReturnButton.tsx

```diff
@@ -1,9 +1,39 @@
 import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
-interface Props {
-  // TODO: define props here
+import { Icon } from 'components/Icon';
+import arrowIcon from 'icons/arrowRight.svg';
+
+const Container = styled.div`
+  a {
+    text-decoration: none;
+    display: flex;
+    align-items: center;
+  }
+
+  img {
+    cursor: pointer;
+  }
+
+  span {
+    font-size: ${({ theme }) => theme.fontSize.s};
+    color: ${({ theme }) => theme.colors.lightDark};
+    margin-left: 10px;
+  }
+`;
+
+export interface Props {
+  content: string;
 }
 
-export const ReturnButton: React.FC<Props> = () => {
-  return <div></div>;
+export const ReturnButton: React.FC<Props> = ({ content }) => {
```

Comment:
- [ ] `Content` to kiepska nazwa - bardzo ogólna i nie wiadomo o co chodzi tak naprawdę - lepiej zmienić na np. `text`.
- [ ] Brakuje przekazania ścieżki, do której ma nastąpić przekierowanie jako props